src/frontend/src/functions/applicationstatusinfo.js

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import {url} from "./url";
+import {checkValidity, getToken, getUsername, setApplicationStatusJson} from "./localstorage";
+import {notification} from "antd";
+
+export async function getApplicationInfo(){
+    if (!checkValidity()) return undefined;
+    const config = {
+        method: 'get',
+        url: url + '/api/students/'+ getUsername() +'/applications',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MarkHershey/SUTDHousingPortal/pullRequest?prid=7298734)
```suggestion
        url: url + "/api/students/"+ getUsername() +'/applications',
```